lib/manager/bazel/extract.js

```diff
@@ -51,7 +51,7 @@ function extractPackageFile(content) {
     let commit;
     let url;
     let sha256;
-    let match = def.match(/name = "([^"]+)"/);
+    let match = def.match(/name\s*=\s*"([^"]+)"/);
```

Comment:
This is a good idea, but these changes ideally should be in a separate refactoring PR. We also need to make sure the bazel update functions also still work as expected in these cases.